Adding storage gloss terms (#445)
redhat-documentation/supplementary-style-guide: supplementary_style_guide/glossary_terms_conventions/general_conventions/d.adoc

*See also*: xref:red-hat-directory-server[Red Hat Directory Server]

[[disaster-recovery]]
==== image:images/yes.png[yes] disaster recovery (adjective)

> *Description*: _Disaster recovery_ refers to processes or methods used to ensure data integrity and the operational continuity of an environment or infrastructure in the event of malicious or accidental interruption.

Reviewer: ```suggestion
*Description*: _Disaster recovery_ refers to processes or methods used to ensure data integrity and the operational continuity of an environment or infrastructure during any malicious or accidental interruption.
```

Reviewer 2: Fine by me.

*See also*: xref:amq-interconnect[AMQ Interconnect]

[[dispersed-volume]]
==== image:images/yes.png[yes] dispersed volume (noun)

> *Description*: A _dispersed volume_ is a Gluster volume that uses erasure coding to write data across three or more bricks to ensure that data remains available even when a certain number of bricks is not available.

Reviewer: Gluster > cluster ?

[.vale-ignore]
*Incorrect forms*: DNF automatic, dnf automatic

[[distributed-file-system]]
==== image:images/yes.png[yes] distributed file system (noun)
*Description*: A _distributed file system_ is a file system that presents files from a number of different storage devices, potentially on many different machines and in many different locations, as a single interface to an end user or consuming service.

*Use it*: yes

[.vale-ignore]
*Incorrect forms*:

*See also*:

[[distributed-volume]]
==== image:images/yes.png[yes] distributed volume (noun)
*Description*: A _distributed volume_ is a Gluster volume that distributes data across one or more bricks.

*Use it*: yes

[.vale-ignore]
*Incorrect forms*:

*See also*:

[distributed-dispersed-volume]]
==== image:images/yes.png[yes] distributed-dispersed volume (noun)

> *Description*: A _distributed dispersed volume_ is a Gluster volume that distributes and erasure codes data across one or more bricks.

Reviewer 3: +1 hyphenation as it's a combination of two modes of data storage

> *Description*: A _distributed dispersed volume_ is a Gluster volume that distributes and erasure codes data across one or more bricks.

Reviewer 2: +1 on hyphenation.
